content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -104,6 +104,33 @@ In this app, you are using the `react-native-global-props` library to set a cust
 
 Now, if you execute `react-native run-ios` or `react-native run-android` you should see the text with your custom fonts.
 
+![IOS](images/customize_fonts_result.png)
+
+## iOS "Unrecognized font family" error
+
+For some of you, maybe you are done (it used to work a few months ago), but if you are still getting the very common **Unrecognized font family** error, there are still some things you can check or do.
```

Comment:
@ManuViola The first part sounds weird, the `For some of you, maybe you are done (it used to work a few months ago),`, how would you rephrase it?

Reply:
mmm, maybe deleting it? something like: 

"After following all these steps, if you are getting the very common..." (and I think I should delete that still because I don't know if the user got that error before, and I use still again later in the sentence) WDYT?

Reply:
Nice! yeah 🙂